hibernate-reactive-core/src/test/java/org/hibernate/reactive/DefaultPortJUnit5Test.java

```diff
@@ -0,0 +1,64 @@
+/* Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * SPDX-License-Identifier: LGPL-2.1-or-later
+ * Copyright: Red Hat Inc. and Hibernate Authors
+ */
+package org.hibernate.reactive;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+import org.hibernate.reactive.containers.DatabaseConfiguration;
+import org.hibernate.reactive.pool.impl.DefaultSqlClientPoolConfiguration;
+import org.hibernate.reactive.provider.Settings;
+
+import org.junit.jupiter.api.Test;
+
+import io.vertx.sqlclient.SqlConnectOptions;
+import org.assertj.core.api.Assertions;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.hibernate.reactive.containers.DatabaseConfiguration.dbType;
+
+/**
+ * Test the default port is set correctly when using {@link DefaultSqlClientPoolConfiguration}
+ */
+public class DefaultPortJUnit5Test {
```

Comment:
We don't need this test, I've added it to check if we could test both version of JUnit, but we already have a `DefaultPortTest`

Reply:
k. np